src/App.tsx

```diff
@@ -1,26 +1,310 @@
+              currentTodos.filter(currentTodo => currentTodo.id !== todo.id),
+            );
+          })
+          .catch(() => {
+            setErrorCompetedDelete(true);
+
+            setTimeout(() => {
+              setErrorCompetedDelete(false);
+            }, 3000);
+          })
+          .finally(() => {
+            setIsInputDisabled(false);
+          });
+      }
+    }
+  };
+
+  useEffect(() => {
+    if (!isInputDisabled && inputRef.current) {
+      inputRef.current.focus();
+    }
+  }, [isInputDisabled]);
+
   if (!USER_ID) {
     return <UserWarning />;
   }
 
   return (
-    <section className="section container">
-      <p className="title is-4">
-        Copy all you need from the prev task:
-        <br />
-        <a href="https://github.com/mate-academy/react_todo-app-loading-todos#react-todo-app-load-todos">
-          React Todo App - Load Todos
-        </a>
-      </p>
-
-      <p className="subtitle">Styles are already copied</p>
-    </section>
+    <div className="todoapp">
+      <h1 className="todoapp__title">todos</h1>
+
+      <div className="todoapp__content">
+        <header className="todoapp__header">
+          {/* this button should have `active` class only if all todos are completed */}
+          <button
+            type="button"
+            className={classNames('todoapp__toggle-all', {
+              active: todos.every(todo => todo.completed),
+              'is-hidden': !todos.length,
+            })}
+            data-cy="ToggleAllButton"
+          />
+
+          {/* Add a todo on form submit */}
+          <form onSubmit={handleSubmit}>
+            <input
+              data-cy="NewTodoField"
+              type="text"
+              className="todoapp__new-todo"
+              placeholder="What needs to be done?"
+              ref={inputRef}
+              value={title}
+              onChange={event => setTitle(event.target.value)}
+              disabled={isInputDisabled}
+            />
+          </form>
+        </header>
+
+        <TodoList
+          todos={filteredTodos}
+          onDeleteTodo={handleDeleteTodo}
+          tempTodo={tempTodo}
+          idDeletedTodo={idDeletedTodo}
+        />
+
+        {/* Hide the footer if there are no todos */}
+        {!todos.length || (
+          <footer className={classNames('todoapp__footer')} data-cy="Footer">
+            <span className="todo-count" data-cy="TodosCounter">
+              {todosLeft.length} items left
+            </span>
+
+            {/* Active link should have the 'selected' class */}
+            <nav className="filter" data-cy="Filter">
+              <a
+                href="#/"
+                className={classNames('filter__link', {
+                  selected: filterType === FilterType.all,
+                })}
+                data-cy="FilterLinkAll"
+                onClick={() => {
+                  const selectedValue = 'all' as FilterType;
+
+                  setFilterType(selectedValue);
```

Comment:
```suggestion
                  setFilterType(FilterType.all);
```

The same in all places